src/utils/openai.ts

```diff
@@ -102,6 +103,13 @@ const deduplicateMessages = (array: string[]) => Array.from(new Set(array));
 
 const getPrompt = (locale: string, diff: string) => `Write a git commit message in present tense for the following diff without prefacing it with anything. Do not be needlessly verbose and make sure the answer is concise and to the point. The response must be in the language ${locale}:\n${diff}`;
 
+const getTokens = (prompt: string, model: TiktokenModel) => {
+	const encoder = encoding_for_model(model);
+	const tokens = encoder.encode(prompt).length;
+	encoder.free();
```

Comment:
It's recommended to free the encoder after the use
https://www.npmjs.com/package/@dqbd/tiktoken